src/containers/orgs/OrgContainer.js

```diff
@@ -476,18 +559,68 @@ class OrgContainer extends Component<Props, State> {
               )
           }
         </div>
-        <div>
-          {
-            memberCardSegments.isEmpty()
-              ? (
-                <i>No members</i>
-              )
-              : (
+      </SectionGrid>
+    );
+  }
+
+  renderMembersSection = () => {
+
+    const { org, memberSearchResults, requestStates } = this.props;
+    const { memberSearchQuery } = this.state;
+
+    const members = org.get('members', List());
+    const memberCardSegments = members.map((member :Map) => (
+      <CompactCardSegment key={member.getIn(['profile', 'user_id'])}>
+        <span>{getUserProfileLabel(member)}</span>
+        {this.renderRemoveButton(() => {
+          this.handleOnClickRemoveMember(member.getIn(['profile', 'user_id']));
+        })}
+      </CompactCardSegment>
+    ));
+
+    const shouldShowSpinner = isNonEmptyString(memberSearchQuery)
+      && requestStates[SEARCH_MEMBERS_TO_ADD_TO_ORG] === RequestStates.PENDING;
+
+    const searchResultCardSegments = memberSearchResults.valueSeq().map((member :Map) => (
+      <CompactCardSegment key={member.get('user_id')}>
+        <span>{getUserProfileLabel(member)}</span>
+        {this.renderAddButton(() => {
+          this.handleOnClickAddMember(member.get('user_id'));
+        })}
+      </CompactCardSegment>
+    ));
+
+    return (
+      <SectionGrid>
+        <h2>Members</h2>
+        <h4>{MEMBERS_SUB_TITLE}</h4>
+        <InputWithButtonWrapper>
+          <SearchInput
+              onChange={this.handleOnChangeMemberSearch}
+              placeholder="Add new member (search by name)" />
+          <Button isLoading={shouldShowSpinner} onClick={this.handleOnClickMemberSearch}>
+            <FontAwesomeIcon icon={faSearch} />
+          </Button>
+        </InputWithButtonWrapper>
+        {
+          isNonEmptyString(memberSearchQuery) && !searchResultCardSegments.isEmpty()
+            ? (
+              <div>
+                <Card>{searchResultCardSegments}</Card>
+              </div>
+            )
+            : null
+        }
+        {
+          !memberCardSegments.isEmpty()
```

Comment:
Not critical but you can short circuit instead of use a ternary
```
{
  !memberCardSegments.isEmpty()
  && (
    <div>
      <Card>{memberCardSegments}</Card>
    </div>
  )
}
```

Reply:
Thanks!